src/components/Breadcrumb/Breadcrumb.tsx

```diff
@@ -50,7 +50,6 @@ const css = (theme: Theme) => {
         padding: '1px',
         $nest: {
           '& *': {
-            display: 'flex',
```

Comment:
Does **align-items** property works without flex element?

Reply:
Yup it will if `parent` is a flex.. That was the issue here as if element is flex it should use `alignItems: center` and if its a child it should use `alignSelf: center`.

Reply:
Okay, I didn't see the parent element's property. Now, I got it.